src/compiler/msvc.rs

```diff
@@ -516,7 +519,7 @@ pub fn parse_arguments(
                     Argument::Raw(ref val) => {
                         if input_arg.is_some() {
                             // Can't cache compilations with multiple inputs.
-                            cannot_cache!("multiple input files");
+                            cannot_cache!("multiple input files or unknown argument: ", String::from(input_arg.unwrap().to_string_lossy()));
```

Comment:
Please use an `if let Some(input_arg) = input_arg { .. }` to avoid the `unwrap`

Reply:
Nice. I'm new to rust and didn't know about this!